examples/with-remix-thirdpartyemailpassword/app/config/appInfo.tsx

```diff
@@ -0,0 +1,8 @@
+export const appInfo = {
+  // learn more about this on https://supertokens.com/docs/thirdpartyemailpassword/appinfo
+  appName: "supertokens_remix_client",
+  apiDomain: "http://localhost:3000",
+  websiteDomain: "http://localhost:3000",
+  apiBasePath: "/api/auth",
```

Comment:
is it common to have apis in remix have the `/api/` path? Like how it's in nextjs? Or is this done this way cause you took inspiration from nextjs?

Reply:
Well, we do not need that. If remix doesn't have a prefix for all APIs, how does it decide if an incoming request is for an API call or for a frontend UI call?

Reply:
We should not use an `/api` prefix in this case, cause users may have other APIs and then adding this prefix, just for supertokens apis is misleading. Instead, we should use something like `/supertokens` as the apiBasePath.